aries-site/src/pages/templates/dashboards.mdx

```diff
@@ -39,3 +41,15 @@ Consider:
 - less detail
 - fewer colors (ex: show 4 pie chart sections and a 5th for other)
 - larger items and fewer items
+
+## Dashboard layouts
```

Comment:
```suggestion
## Variants
```

Should we remain consistent with some of our other page structures?

Reply:
In this case, I am advocating for "Dashboard Layouts" or "Dashboard Templates" as I think that more accurately describes the underlying contents.The idea is that we are building a collection of templates folks can select from. 

"Variants" has been used more in the context of Components where we are showcasing different properties of the components. Within Templates, we have not been using "Variants" all that much. The headings seem to be more aligned with the narrative of the template subject.

Examples where we are not using the term "variants":
- https://design-system.hpe.design/templates/filtering
- https://design-system.hpe.design/templates/forms
- https://design-system.hpe.design/templates/content-layouts#templates
- https://design-system.hpe.design/templates/toast-notifications
- https://design-system.hpe.design/templates/status-indicator
- https://design-system.hpe.design/templates/datatable-customization
- https://design-system.hpe.design/templates/code-blocks
- https://design-system.hpe.design/templates/navigation
- https://design-system.hpe.design/templates/page-layouts


Examples using "variants":
- https://design-system.hpe.design/templates/lists 
- https://design-system.hpe.design/templates/wizard 

After peeking around, 

I am tempted to go "Dashboard Templates" because a) its an accurate descriptor, and b) folks will be linking to it from https://design-system.hpe.design/templates/content-layouts#templates.

Reply:
I'm late to the party but want to give my support for 'Dashboard Templates' given...

> a) its an accurate descriptor, and b) folks will be linking to it from https://design-system.hpe.design/templates/content-layouts#templates.

Good call, all!

Reply:
Cool, I'm good with this.